composer.json

```diff
@@ -48,7 +48,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.3.x-dev"
+            "dev-master": "2.0.x-dev"
```

Comment:
the release 1.3.4 should be the last in the 1.x line. master already has a new feature with the priorities, but i missed to update the branch alias.